Runtime/Core/LensSettings.cs

```diff
@@ -324,6 +324,7 @@ public static LensSettings Lerp(LensSettings lensA, LensSettings lensB, float t)
         /// <summary>Make sure lens settings are sane.  Call this from OnValidate().</summary>
         public void Validate()
         {
+            NearClipPlane = Mathf.Max(NearClipPlane, 0.01f);
```

Comment:
We took this out on purpose, you were part of that decision.  This check is only valid if not orthographic.  You cannot check for that here.  If you like, you can add a comment so that this line does not return in the future.

Reply:
Oh yes, I forgot thanks for reminding me!